src/containers/StructurePage/resourceComponents/StructureResources.jsx

```diff
@@ -13,13 +13,17 @@ import { spacing } from '@ndla/core';
 import styled from '@emotion/styled';
 import ResourceGroup from './ResourceGroup';
 import AllResourcesGroup from './AllResourcesGroup';
-import { groupSortResourceTypesFromTopicResources } from '../../../util/taxonomyHelpers';
+import {
+  groupSortResourceTypesFromTopicResources,
+  getIdFromUrn,
+} from '../../../util/taxonomyHelpers';
 import { fetchAllResourceTypes, fetchTopicResources, fetchTopic } from '../../../modules/taxonomy';
 import handleError from '../../../util/handleError';
 import TopicDescription from './TopicDescription';
 import Spinner from '../../../components/Spinner';
 import { fetchDraft } from '../../../modules/draft/draftApi';
 import { fetchLearningpath } from '../../../modules/learningpath/learningpathApi';
+import { getArticle } from '../../../modules/article/articleApi';
```

Comment:
Treng du denne? Du har vel allerede infoen du trenger i fra getArticle der du henter draft.

Reply:
Jeg bytta ut denne med fetchDraft, men vi må likevel gjøre de ekstra kallene siden getArticle bare kalles for emneartikkelen

Reply:
Men hentes ikkje draft for alle ressurser i getResourceStatuses i StructureResources? Kan du ikkje gjøre noko med den?